UNRELEASED.md

```diff
@@ -8,11 +8,12 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
-Updated `Textfield` with a `type` of `number` to not render a spinner if step is set to `0` ([#3477](https://github.com/Shopify/polaris-react/pull/3477))
+- Updated Textfield with a type of number to not render a spinner if step is set to 0 ([#3477](https://github.com/Shopify/polaris-react/pull/3477))
```

Comment:
This can be 🔥  right? Should be captured in changelog already

Reply:
It hasn't been released so it can stay until we release an actual version ( not a beta )